remoteworkitem/remoteworkitem_test.go

```diff
@@ -75,160 +75,238 @@ func TestWorkItemMapping(t *testing.T) {
 func TestGitHubIssueMapping(t *testing.T) {
 	resource.Require(t, resource.UnitTest)
 
-	content, err := test.LoadTestData("github_issue_mapping.json", provideRemoteGithubDataWithAssignee)
-	if err != nil {
-		t.Fatal(err)
+	type githubData struct {
+		inputFile      string
+		expectedOutput bool
 	}
 
-	workItemMap := WorkItemKeyMaps[ProviderGithub]
-	remoteTrackerkItem := TrackerItem{Item: string(content[:]), RemoteItemID: "xyz", TrackerID: uint64(0)}
+	var gitData = []githubData{
+		{"github_issue_mapping.json", true},
```

Comment:
That each Field in the mapped workItem return 'true' is a bit too simplistic of a test. I'm sure we have scenarios for optional data etc as well.